templates/repo/graph/commits.tmpl

```diff
@@ -34,7 +34,8 @@
 							{{$refGroup := .RefGroup}}
 							{{if eq $refGroup "pull"}}
 								{{if or (not $.HidePRRefs) (containGeneric $.SelectedBranches .Name)}}
-									<a class="ui labelled icon button basic tiny" href="{{$.RepoLink}}/pulls/{{.ShortName|PathEscape}}">
+									<!-- it's intended to use issues not pulls, it it's a pull you will get redirected -->
```

Comment:
```suggestion
									<!-- it's intended to use issues not pulls, if it's a pull you will get redirected -->
```

Reply:
if you have better comment, just suggest :)

Reply:
the first part of the sentence could just state that "use of 'issues', not 'pulls' here is intended" and perhaps links to this thread could be added, but I's fine.